README.md

```diff
@@ -267,6 +267,9 @@ let counterClone = Counter.fromHex(counter.lockingScript.toHex());
 
 ```
 
+## Chain Api
+
+[Chain Api](./docs/chain_api_en.md) makes building transactions more efficient and simple
```

Comment:
makes building transactions more efficient and simple
->
makes building transactions easy.